tools/fastg2protlib/fastg2protlib-peptides.xml

```diff
@@ -0,0 +1,65 @@
+<tool id="fastg2protlib-peptides" name="FASTG2Protlib-Peptides" version="@VERSION@">
+    <macros>
+        <import>macros.xml</import>
+    </macros>
+    <description>Generate a peptide FASTA library starting with a FASTG assembly file.</description>
+
+    <requirements>
+        <requirement type="package" version="@VERSION@">fastg2protlib</requirement>
+    </requirements>
+    
+    <command detect_errors="exit_code">
+        <![CDATA[
+        python $__tool_directory__/application.py 
+        -m $min_peptide_length
+        -p $min_protein_length  
+        -c $cleavage 
+        -d $database_name 
+        -l $show_plots
+        $fastg_file
+        ]]>
+    </command>
+    <inputs>
+        <param name="fastg_file" type="data" format="fastg" label="FASTG file" />
+        <param name="cleavage" type="select" label="Peptide Cleavage">
+            <option value="trypsin" selected="true">Trypsin</option>
+            <expand macro="cleavages" />
+        </param>
+        <param name="min_protein_length" type="integer" value="55" label="Minimum Protein Length in Amino Acids" />
+        <param name="min_peptide_length" type="integer" value="8" label="Minimum Peptide Length in Amino Acids" />
+        <param name="database_name" type="text" value="results.db" label="Database Name" />
```

Comment:
where does this come from? Does this need to exposed to the user?

Reply:
peptide, protein amino acid lengths are a user defined param since lengths can vary based on MSMS type, species and enzymatic cleavage used. Length is a sanity check on validity of results.
DB name is open to user  decision since it is the user's data.

Reply:
`database_name` should (must?) not be exposed to the user. If I get the code right its used as the filename of the sqlite DB.

Reply:
Changed